web/apps/photos/src/components/Sidebar/UtilitySection.tsx

```diff
@@ -72,6 +79,26 @@ export default function UtilitySection({ closeSidebar }) {
         closeSidebar();
 
         try {
+            // check if the user has passkey recovery enabled
+            const recoveryEnabled = await isPasskeyRecoveryEnabled();
+            if (!recoveryEnabled) {
+                // let's create the necessary recovery information
+                const recoveryKey = await getRecoveryKey();
+
+                const resetSecret = uuidv4();
+
+                const encryptionResult = await encryptToB64(
+                    resetSecret,
+                    recoveryKey,
+                );
+
+                await configurePasskeyRecovery(
+                    resetSecret,
+                    encryptionResult.encryptedData,
+                    encryptionResult.nonce,
+                );
+            }
+
             const accountsToken = await getAccountsToken();
 
             window.location.href = `${getAccountsURL()}${
```

Comment:
Can we open accounts.ente.io in a separate tab? If yes, let's please switch to that..